public/index.html

```diff
@@ -432,7 +487,23 @@ <h5 class="font-semibold text-xl md:text-2xl pb-2">Ending</h5>
       </div>
     </div>
     -->
-
+    <section class="bg-waves-bg-upsd aspect-video w-full bg-repeat bg-center bg-cover p-200">
+      <h3 class="text-5xl lg:text-6xl text-center text-white pt-20">Sponsors</h3>
+      <div class="mx-10">
+        <h2 class="text-3xl md:text-4xl lg:text-5xl text-center font-semibold pt-10 text-white pb-6">Gold</h2>
+      </div>
+    
+    <div class="flex justify-center	">
+        <div class="grid grid-cols-1 divide-y-2 w-4/5">
+          <div></div>
+          <div class="py-16 flex justify-center">
+              <img class="px-10 md:h-2/5 lg:h-2/6 hover:scale-110 transition-all ease-linear transition ease-in-out delay-125  hover:grayscale-0 grayscale-40 hover:shadow-xl" src="img/scroll.svg" alt="scroll">
```

Comment:
add URL link

Reply:
and the logotype has too little contrast on black bg - use another logo version or use another bg colour?